web/src/main/java/se/inera/intyg/webcert/web/web/controller/facade/dto/ResourceLinkDTO.java

```diff
@@ -45,6 +46,18 @@ public static ResourceLinkDTO create(ResourceLinkTypeDTO type, String name, Stri
         return resourceLink;
     }
 
+    public static ResourceLinkDTO create(ResourceLinkTypeDTO type, String name, String description, String body, String receiver,
+        boolean enabled) {
+        final var resourceLink = new ResourceLinkDTO();
+        resourceLink.setType(type);
+        resourceLink.setName(name);
+        resourceLink.setDescription(description);
+        resourceLink.setBody(body);
+        resourceLink.setEnabled(enabled);
+        resourceLink.setReceiver(receiver);
```

Comment:
I think we have to look for a different way to provide the receiver to the client. Have a field in the more "generic" ResourceLink is specific to sending certificate and will not be used by any other type of ResourceLink.